third_party/rake-compiler-dock/README.md

```diff
@@ -0,0 +1,46 @@
+rake-compiler-dock for gRPC
+===================================
+
+This has customized docker images for Ruby based on
+[rake-compiler-dock](https://github.com/rake-compiler/rake-compiler-dock).
+gRPC needs four docker images to build Ruby artifacts;
+
+- rake_x64-linux: Linux / 32bit
+- rake_x86_64-linux: Linux / 64bit
+- rake_x86-mingw32: Windows / 32bit
+- rake_x64-mingw32: Windows / 64bit
+
+### Customization
+
+#### Linux
+
+The linux docker images of `rake-compiler-dock` are based on Ubuntu, which is enough for
```

Comment:
Can we please also document the motivation for all of the `.patch` files written here?

Reply:
Those files came directly from `rake-compiler-dock` so it's not necessary to explain what they're for.